Linux/papercut/config.properties

```diff
@@ -1,216 +1,4 @@
-# 
-# Example: default-account-pin=1122
-#
-#default-account-pin=
-
-# Authentication time-to-live options that override settings from server
-# Time-to-live values in minutes (Comma separated) 
-#auth-ttl-values=1,5,15,30,60,-1
-#
-#auth-ttl-default=5
-
-# The "disable-gui-effects" option instructs the client to turn off
-# some of the advanced GUI effects such as transparent windows and
-# fading images.  This may improve performance on very old systems.
-#
-#disable-gui-effects=Y
-
-# The "lockdir" option defines the directory where the client lock
-# file is created.  The lock file is used by the client to ensure that
-# each user only runs the client once.  The lock files are usually stored
-# in the user's home directory with file names matching the format:
-#      .uc-<hash>.<username>.<workstation>.tmp
-#
-# NOTE: User '\\' for path separators on Windows
-#
-# Example: lockdir=C:\\temp\\
-#
-#lockdir=
-
-# The "disable-auth-by-id-number" option may be specified to revert the
-# authentication method to user/password instead of ID Number.
-#
-#disable-auth-by-id-number=Y
-
-# The "use-strict-ssl-verification" is used to enable strict checking of SSL
-# certificates used to connect to the application server.  This includes both
-# hostname and certificate authority trusts.
-#
-# If enabling this option you MUST have an officially signed certificate
-# installed on the application server, and use the server's DNS name in both
-# the "server-ip" and "server-name" settings.
-#
-#use-strict-ssl-verification=Y
-
-# The following options define the text to display in client windows.  These
-# can be useful if you wish to customize the text to suit your environment.
-#
-#account-from-list-allocate-text=Allocate to shared account
-#account-from-list-text=Charge to shared account
-#account-pin-text=PIN / Code
-#account-with-pin-allocate-text=Allocate to shared account using PIN / Code
-#account-with-pin-text=Charge to shared account using PIN / Code
-#auth-description-text=Your login credentials are required for printer access.
-#auth-login-text=Login
-#auth-password-text=Password:
-#auth-username-text=Username:
-#auth-pin-text=PIN:
-#auth-id-number-text=ID Number:
-#identity-description-text=Your login credentials are required for printer access.
-#identity-detailed-text=Please log in using your network username and password to confirm your identity.
-#identity-login-text=Login
-#identity-password-text=Password:
-#identity-username-text=Username:
-#no-accounts-text=No shared accounts available
-#other-user-account-text=Perform print as user
-#personal-account-text=Charge to my personal account
-#select-account-text=Select the shared account to charge ...
-#user-balance-for-text=Balance for {0}
-#user-close-warning-text=The application cannot be closed, because it is required to perform printing on this network.
-#user-details-text=Details ...
-
+#Bootstrap configuration information
+#Tue May 22 09:21:01 EDT 2018
+server-port=9191

```

Comment:
Looks like this is a [carriage return](https://en.wikipedia.org/wiki/Carriage_return) - Windows uses CRLF to designate an endline while linux just uses LF.  This shouldn't how the script runs.

Reply:
sounds good